src/app/collective-rewards/rewards/AllTimeRewards.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  formatMetrics,
+  MetricsCardTitle,
+  MetricsCardWithSpinner,
+  TokenMetricsCardRow,
+  useGetBuilderRewards,
+  useGetBuilderRewardsClaimedLogs,
+} from '@/app/collective-rewards/rewards'
+import { useHandleErrors } from '@/app/collective-rewards/utils'
+import { formatBalanceToHuman } from '@/app/user/Balances/balanceUtils'
+import { usePricesContext } from '@/shared/context/PricesContext'
+import { FC, useEffect, useState } from 'react'
+import { Address } from 'viem'
+
+type Token = {
+  symbol: string
+  address: Address
+}
+
+type TokenRewardsProps = {
```

Comment:
I need to change this

Reply:
I changed the component name instead as it made more sense to me